### DISCUSSION TITLE
Can we access a +page.server.ts endpoint without the .svelte-page like before (when we used __data.json)

### DISCUSSION THREAD

**Discussion by lkj4** [sveltejs/kit#6665] [Q&A]
So, I have this former shadow endpoint, now a +page.server.ts which works fine. How can I browser-fetch against this endpoint without loading the .svelte-file. This was possible before.

**Comment by teemingc** [comment]
For now, you'll have to create a `GET` endpoint in a `+server.ts` file.

**Reply by lkj4** [reply]
ok and good to know but this is not DRY at all. is it because the response is serialized with devalue? and if yes, how can I deserialize it myself?

and will it change or stay like this?

**Reply by teemingc** [reply]
> ok and good to know but this is not DRY at all. is it because the response is serialized with devalue? and if yes, how can I deserialize it myself?
> 
> and will it change or stay like this?

You can follow the conversation on why it worked before and was removed in these issues #5936 #6007

May I ask what your use case is?

**Reply by lkj4** [reply]
> May I ask what your use case is?

sure, I've just posted it into https://github.com/sveltejs/kit/issues/5896#issuecomment-1240885789
